wolfcrypt/src/fe_448.c

```diff
@@ -1437,56 +1437,56 @@ void fe448_to_bytes(unsigned char* b, const sword32* a)
     b[ 0] = (byte)(in0  >>  0);
     b[ 1] = (byte)(in0  >>  8);
     b[ 2] = (byte)(in0  >> 16);
-    b[ 3] = (byte)(in0  >> 24) + (byte)((in1  >>  0) <<  4);
+    b[ 3] = (byte)((byte)(in0  >> 24) + (byte)((in1  >>  0) <<  4));
```

Comment:
Can we not do:
```
    b[ 3] = (byte)((in0 >> 24) + (in1 >> 0) << 4));
```
???

Reply:
That would _probably_ give bitwise-identical results on all targets, however I removed all doubt by leaving the existing pre-addition casts in place, and just adding an outer cast.  Was just being expeditious.